src/invidious/yt_backend/extractors.cr

```diff
@@ -45,13 +59,16 @@ private module Parsers
   # `videoRenderer`s can be found almost everywhere on YouTube. In categories, search results, channels, etc.
   #
   module VideoRendererParser
+    extend self
+    include BaseParser
+
     def self.process(item : JSON::Any, author_fallback : AuthorFallback)
```

Comment:
You can also remove `self` here
```suggestion
    def process(item : JSON::Any, author_fallback : AuthorFallback)
```

Note: please keep the `self` in `return self.parse(...)`, as it's easier to understand that we're using a module method and rathern than an external function.